doc/BOOTSTRAP.md

```diff
@@ -0,0 +1,107 @@
+# The Horde 5 Bootstrapping Process
```

Comment:
The documentation belongs to Wiki, which should be the leading source. We have tooling in horde-components to pull docs from there into reST files for distribution. See Argv for an example.